crypto/Cargo.toml

```diff
@@ -12,7 +12,7 @@ authors = ["Samer Afach <samer.afach@mintlayer.org>", "Ben Marsh <benjamin.marsh
 serialization = { path = "../serialization" }
 secp256k1 = { version = "0.24.2", default-features = false, features = ["global-context", "rand-std"] }
 hmac = "0.12.1"
-bip39 = { version = "1.0.1", default-features = false }
+bip39 = { version = "1.0", default-features = false }
```

Comment:
Maybe do the same for `hmac` :smile:

Btw, I see there's a new version for secp256k1 crate.